src/components/SkillCard/SkillCard.scss

```diff
@@ -0,0 +1,25 @@
+@use '../../sass/abstracts/variables';
+
+.skillCard {
+  background-size: cover;
+  min-width: 300px;
+  min-height: 300px;
+  position: relative;
+  border-radius: variables.$fs-6;
+}
+
+.skillCard__image {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+}
+
+.skillCard__text {
+  position: absolute;
```

Comment:
you can use flex-box and `margin-top: auto` for this.

Reply:
Interesting play with flex-box. Took me some reading to figure out..✅